massgov/mayflower
# Update sticky TOC to only target h2 elements

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

The current version of Mayflower targets all comp headings, not just h2 elements. This limits it to h2 elements.

## Related Issue / Ticket

- [DP-8019](https://jira.mass.gov/browse/DP-8019)

## Steps to Test


1. Checkout the branch.
2. Confirm that on the following pages, the sticky TOC is properly working and only pulling h2 elements:
    * http://localhost:3000/patterns/05-pages-guide/05-pages-guide.html
    * http://localhost:3000/patterns/05-pages-information-details-table-of-contents/05-pages-information-details-table-of-contents.html

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
